libminifi/test/keyvalue-tests/PersistableKeyValueStoreServiceTest.cpp

```diff
@@ -33,17 +32,14 @@
 
 static std::string config_yaml; // NOLINT
 
-static inline void configYamlHandler(Catch::ConfigData&, const std::string& path) {
-  config_yaml = path;
-}
-
 int main(int argc, char* argv[]) {
   Catch::Session session;
 
-  auto& cli = const_cast<Catch::Clara::CommandLine<Catch::ConfigData>&>(session.cli());
-  cli["--config-yaml"]
-      .describe("path to the config.yaml containing the PersistableKeyValueStoreService controller service configuration")
-      .bind(&configYamlHandler, "path");
+  auto cli = session.cli()
+      | Catch::clara::Opt{config_yaml, "config-yaml"}
```

Comment:
I would put the `|` at the end of the line.  It's OK at the beginning of the next line, too, but then please do the same in UnorderedMapKeyValueStoreServiceTest.cpp.  Seeing it in two different ways triggers my OCD. :smile:

Reply:
Whoops, I missed that. Moved the other one to the beginning of the next line. I like that style better because if there are multiple operations, they are nicely lined up and not "hidden" at the end of the line.